src/main/java/blusunrize/immersiveengineering/common/items/HammerItem.java

```diff
@@ -171,7 +171,7 @@ public ActionResultType onItemUseFirst(ItemStack stack, ItemUseContext context)
 				return ActionResultType.SUCCESS;
 			else if(!rotate&&tile instanceof IHammerInteraction)
 			{
-				if(((IHammerInteraction)tile).hammerUseSide(side, player, context.getHitVec()))
+				if(((IHammerInteraction)tile).hammerUseSide(side, player, context.getHand(), context.getHitVec()))
```

Comment:
Any particular reason for this change? I don't think I saw the new parameter used anywhere.

Reply:
Posts. They needed to create an ItemUseContext to make an isReplacable check. That context needs a hand, which wasn't being passed down.
Also, this may come in useful in the future.